src/js/components/projects/listItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Product } from '@/store/products/reducer';
+import { Project } from '@/store/projects/reducer';
+import routes from '@/utils/routes';
+
+interface Props {
+  product: Product;
+  project: Project;
+}
+
+const ProjectListItem = ({ product, project }: Props) => {
```

Comment:
@duggiemitchell Can we make this into a list? I don't know where hte <ul> would go here since this is a repeated block and I only want one ul wrapper element, but I'm looking at something like: 
```
<ul className="slds-has-dividers_bottom-space">
    <li className="slds-item project-list-item">
      {/* @todo capture slug from actual project */}
      <h2 className="slds-text-heading_medium">
        <Link to={routes.project_detail(product.slug, slug)}>
          {name}
        </Link>
      </h2>
      {description && (
        <p
          className="markdown project-list-description"
          dangerouslySetInnerHTML={{ __html: description }}
        />
      )}
      </li>
  </ul> 
```